src/generate_pisa_index.cpp

```diff
@@ -4,12 +4,32 @@
 #include "google/protobuf/io/coded_stream.h"
 #include <google/protobuf/io/zero_copy_stream.h>
 #include <google/protobuf/io/zero_copy_stream_impl.h>
+#include <google/protobuf/util/delimited_message_util.h>
 #include "common-index-format.pb.h"
 #include "CLI/CLI.hpp"
 #include "gsl/span"
 
+using namespace cif;
 
+void dump_postings_list(const PostingsList& postings_list) {
+    std::cerr << "Term = [" << postings_list.term() << "]" << std::endl;
+    std::cerr << "Document Frequency/Collection Term Frequency = [" 
+              << postings_list.df() << "," << postings_list.cf() << "]" 
+              << std::endl;
+   
+    int32_t prev_id = 0; 
+    for (int64_t i = 0; i < postings_list.posting_size(); ++i) {
+        const Posting& posting = postings_list.posting(i);
+        int32_t doc_id = prev_id + posting.docid();
+        std::cerr << "[" << doc_id << "," << posting.tf() << "] ";
+        prev_id = doc_id;
+    }
+    std::cerr << std::endl; 
+}
 
+void dump_term(const PostingsList& postings_list, uint32_t term_id) {
+  std::cerr << postings_list.term() << " " << term_id << " " << postings_list.df() << "\n";
+}
```

Comment:
They don't seem to be used anywhere, or am I missing something?

Reply:
Just for debugging - we can remove them, but they may come in useful for other users of the program. I kept them just in case.